graylog2-server/src/main/java/org/graylog2/rest/resources/streams/StreamResource.java

```diff
@@ -111,14 +112,30 @@
 @Api(value = "Streams", description = "Manage streams", tags = {CLOUD_VISIBLE})
 @Path("/streams")
 public class StreamResource extends RestResource {
-    private static final Logger LOG = LoggerFactory.getLogger(StreamResource.class);
-
-    protected static final ImmutableMap<String, SearchQueryField> SEARCH_FIELD_MAPPING = ImmutableMap.<String, SearchQueryField>builder()
-            .put("id", SearchQueryField.create(StreamDTO.FIELD_ID, SearchQueryField.Type.OBJECT_ID))
-            .put(StreamDTO.FIELD_TITLE, SearchQueryField.create(StreamDTO.FIELD_TITLE))
-            .put(StreamDTO.FIELD_DESCRIPTION, SearchQueryField.create(StreamDTO.FIELD_DESCRIPTION))
+    private static final String DEFAULT_SORT_FIELD = StreamDTO.FIELD_TITLE;
+    private static final String DEFAULT_SORT_DIRECTION = "asc";
+    private static final List<EntityAttribute> attributes = List.of(
+            EntityAttribute.builder().id("title").title("Title").build(),
```

Comment:
type has no default, but it looks as "string" is considered a default - because you set "date", "boolean" - but not "string". is this on purpose or an oversight?

Reply:
This is on purpose. If no `type` is defined for a field, we do assume it is text(-like).